.github/workflows/release-embedding-sdk.yml

```diff
@@ -151,13 +202,13 @@ jobs:
 
       - name: Create a PR updating readme + published version, and changelog (using GitHub Metabase Automation account)
         run: |
-          git checkout -b update-sdk-version-${{ inputs.sdk_version }}
-          git commit -a -m 'Update Readme version references and published npm version to ${{ inputs.sdk_version }}'
+          git checkout -b update-sdk-version-${{ env.sdk_version }}
+          git commit -a -m 'Update Readme version references and published npm version to ${{ env.sdk_version }}'
           git push origin HEAD
-          gh pr create --base master\
+          gh pr create --base ${{ inputs.branch }}\
```

Comment:
now we base the branch on the current branch being deploy because `master` and the `release` branch will have different changelogs since they have different version numbers.